tests/test_ebs.py

```diff
@@ -280,3 +280,17 @@ def test_ebs_metrics_percent_filter(self):
             }, session_factory=session)
         resources = policy.run()
         self.assertEqual(len(resources),1)
+
+class HealthEventsFilterTest(BaseTest):
+    def test_ebs_health_events_filter(self):
+        session_factory = self.replay_flight_data(
+            'test_ebs_health_events_filter')
+        policy = self.load_policy({
+            'name': 'ebs-health-events-filter',
+            'resource': 'ebs',
+            'filters': [{
+                'type': 'health-event',
+                'types': ['AWS_EBS_VOLUME_LOST']}]
+                },session_factory=session_factory)
+        resources = policy.run()
+        self.assertEqual(len(resources), 0)
```

Comment:
we probably want to do record this against a place where it actually exhibits and then manually cleanup the recorded files to get a positive assertion test.